collectives/routes/auth.py

```diff
@@ -60,17 +76,25 @@ def sync_user(user, force):
 
 
 def create_confirmation_token(license_number, user):
+    """ Create a token for email confirmation.
+
+    :return: New confirmation token saved into database.
+    :rtype: :py:class:`collectives.models.auth.ConfirmationToken`
+    """
     token = ConfirmationToken(license_number, user)
     db.session.add(token)
     db.session.commit()
     return token
 
 
-##########################################################################
-#   LOGIN
-##########################################################################
 @blueprint.route("/login", methods=["GET", "POST"])
 def login():
+    """ Route for user login page.
+
+    If authentification is failed, there is a timeout : :py:data:`config.AUTH_FAILURE_WAIT`
+    If confidentiality agreement or legal text has not been signed, a reminder is
+    sent to user if login is succesfull.
```

Comment:
Typo 'successful' (et 'has failed' plutot que 'is failed'). Peut être aussi 'shown' plutôt que 'sent' to the user)